LabTools/CoreWidgets/PropertiesWidget.py

```diff
@@ -0,0 +1,756 @@
+# -*- coding: utf-8 -*-
```

Comment:
@zackorenberg - As this one is not working yet, I would copy it outside and add it in a subsequent PR. Would you agree ?

Reply:
This is working perfectly on my computer with a few devices, include A33220A and SIM900